examples/fcontext.pp

```diff
@@ -1,4 +1,4 @@
 selinux::fcontext{'set-mysql-log-context':
-  context  => 'mysqld_log_t',
-  pathname => '/var/tmp/mysql',
+  seltype => "mysqld_log_t",
```

Comment:
Single quotes is still the right way. Could you fix this?

Reply:
I just took what's at https://github.com/voxpupuli/puppet-selinux/blob/master/manifests/fcontext.pp#L5 so that anyone can use this example with the master branch and not fail. Would we have to change the example in the manifest as well?

Reply:
Ideally yes. Currently the build is failing on linting but we can't lint examples. That would actually be a nice idea :)

Reply:
I made the change for the example.